### ISSUE TITLE
Release code of co-train？

### ISSUE THREAD

**Issue by fengxiuyaun** [RoboFlamingo/RoboFlamingo#9]

**Comment by lixinghang12** [comment]
The co-train code has been updated, please see `[pt_run_gripper_post_ws_12_traj_aug_mpt_dolly_3b_co_train.bash](https://github.com/RoboFlamingo/RoboFlamingo/blob/main/robot_flamingo/pt_run_gripper_post_ws_12_traj_aug_mpt_dolly_3b_co_train.bash)` for more details.

**Comment by fengxiuyaun** [comment]
Yes, but  "cotrain" is not used

**Comment by lixinghang12** [comment]
@fengxiuyaun Sorry, there's a typeo in line 21 of [pt_run_gripper_post_ws_12_traj_aug_mpt_dolly_3b_co_train.bash](https://github.com/RoboFlamingo/RoboFlamingo/blob/main/robot_flamingo/pt_run_gripper_post_ws_12_traj_aug_mpt_dolly_3b_co_train.bash), i have fix it from `co_train` to `cotrain`.

**Comment by COST-97** [comment]
Hello:
The code for cotrain seems to be incomplete.
1. In train_calvin.py, `train_one_epoch_calvin_cotrain` is not called.
2. The `output.logits` in the flamingo_mpt.py are only the corresponding `lm_head` of the robot tasks, which should not be compatible with coco and VQA data, right?
Could you add to this part of the code?
Thank you!
